integration/helpers/playwright-fixture.ts

```diff
@@ -42,7 +42,7 @@ export class PlaywrightFixture {
       throw new Error(`Could not find link for ${selector}`);
     }
     if (options.wait) {
-      await doAndWait(this.page, () => el.click());
+      await doAndWait(this.page, () => el!.click());
```

Comment:
Hey @ryanflorence, I'm pretty sure if we used `const el` above, we wouldn't have to do this :P

Reply:
Yup, using `const` fixes it (but then I'd have to use `eslint-disable-next-line`). Alternatively I could remove the `!` and add `@ts-expect-error` (until such time as [narrowing is improved](https://github.com/microsoft/TypeScript/issues/35124)). Happy to switch to one of these if we prefer them stylistically.